client/src/components/registration/RegistrationForm.js

```diff
@@ -85,22 +86,32 @@ const RegistrationForm = () => {
   }
 
   return (
+    <div className="form">
+    <div className="trail-bg-img">
     <div className="grid-container" onSubmit={onSubmit}>
```

Comment:
Same as above, you can combine these 3 classes, separated by a space, into the same `className=""`

Reply:
absolutely!